bitbots_vision/src/bitbots_vision/vision_modules/field_boundary.py

```diff
@@ -10,10 +10,22 @@
 
 
 class FieldBoundaryDetector(object):
+    """
+    The abstract class :class:`.FieldBoundaryDetector` is used for detecting the field boundary in various ways.
+    The task of such a detector is the localisation of the edges of the field in the image.
+
+    It returns a list of points that form this so called field boundary.
+    It requires the ColorDetector to find the green pixels that are used to identify the field in the picture.
+    The pixels of the field boundary are found by traversing the picture column wise in steps of a given length.
+    Because obstacles can obscure the edges of the field, sometimes the first green pixel from the top of the picture is found at the bottom of the respective obstacle.
+    Therefore not all of the points are located in a straight line and the field boundary contains multiple dents.
+    Additionally white field markings and green pixels in the field that are false negatives can create small dents too.
+    Besides the normal field boundary, the :class:`.FieldBoundaryDetector` can also create a convex field boundary that forms a convex hull over the dents of the detected field boundary and is therefore completely straight (with the exception of the corners of the field).
```

Comment:
This line seems too long and should be split up into two lines

Reply:
> This line seems too long and should be split up into two lines

```suggestion
    Besides the normal field boundary, the :class:`.FieldBoundaryDetector` can also create a convex field boundary
    that forms a convex hull over the dents of the detected field boundary
    and is therefore completely straight (with the exception of the corners of the field).
```